packages/ui/src/working-groups/components/WorkersList.tsx

```diff
@@ -28,7 +28,12 @@ export const WorkersList = ({ lead, workers }: WorkersListProps) => {
         {workers && (
           <ContentWithTabs>
             {workers.map((worker, index) => (
```

Comment:
Isn't it easier to just not render anything when the worker is a lead? Especially that `isLead` sounds like this component will render smth extra when the worker is a leader, not just omit it.

```suggestion
            {workers.map((worker, index) => worker.member.id !== lead?.meber.id && (
```
or just filter the results

```suggestion
            {workers.filter(worker => worker.member.id !== lead?.member.id).map((worker, index) => (
```